Makefile

```diff
@@ -31,6 +31,7 @@ invalidate-cfn-cache:
 	aws cloudfront create-invalidation --distribution-id $(CLOUDFRONT_DIST_ID) --paths /nlx/*
 
 push-to-auth0: sanity-checks
+	test -e dist/index.html
```

Comment:
should put this in the sanity-checks target, no?

Reply:
Sure thing